source/grid/grid_generator.cc

```diff
@@ -8439,6 +8444,674 @@ namespace GridGenerator
+       {{5, 8, 4, 5, 10, 8, 10, 11, 8, X, X, X, X, X, X, X}},
+       {{5, 0, 4, 5, 11, 0, 5, 10, 11, 11, 3, 0, X, X, X, X}},
+       {{0, 1, 9, 8, 4, 10, 8, 10, 11, 10, 4, 5, X, X, X, X}},
+       {{10, 11, 4, 10, 4, 5, 11, 3, 4, 9, 4, 1, 3, 1, 4, X}},
+       {{2, 5, 1, 2, 8, 5, 2, 11, 8, 4, 5, 8, X, X, X, X}},
+       {{0, 4, 11, 0, 11, 3, 4, 5, 11, 2, 11, 1, 5, 1, 11, X}},
+       {{0, 2, 5, 0, 5, 9, 2, 11, 5, 4, 5, 8, 11, 8, 5, X}},
+       {{9, 4, 5, 2, 11, 3, X, X, X, X, X, X, X, X, X, X}},
+       {{2, 5, 10, 3, 5, 2, 3, 4, 5, 3, 8, 4, X, X, X, X}},
+       {{5, 10, 2, 5, 2, 4, 4, 2, 0, X, X, X, X, X, X, X}},
+       {{3, 10, 2, 3, 5, 10, 3, 8, 5, 4, 5, 8, 0, 1, 9, X}},
+       {{5, 10, 2, 5, 2, 4, 1, 9, 2, 9, 4, 2, X, X, X, X}},
+       {{8, 4, 5, 8, 5, 3, 3, 5, 1, X, X, X, X, X, X, X}},
+       {{0, 4, 5, 1, 0, 5, X, X, X, X, X, X, X, X, X, X}},
+       {{8, 4, 5, 8, 5, 3, 9, 0, 5, 0, 3, 5, X, X, X, X}},
+       {{9, 4, 5, X, X, X, X, X, X, X, X, X, X, X, X, X}},
+       {{4, 11, 7, 4, 9, 11, 9, 10, 11, X, X, X, X, X, X, X}},
+       {{0, 8, 3, 4, 9, 7, 9, 11, 7, 9, 10, 11, X, X, X, X}},
+       {{1, 10, 11, 1, 11, 4, 1, 4, 0, 7, 4, 11, X, X, X, X}},
+       {{3, 1, 4, 3, 4, 8, 1, 10, 4, 7, 4, 11, 10, 11, 4, X}},
+       {{4, 11, 7, 9, 11, 4, 9, 2, 11, 9, 1, 2, X, X, X, X}},
+       {{9, 7, 4, 9, 11, 7, 9, 1, 11, 2, 11, 1, 0, 8, 3, X}},
+       {{11, 7, 4, 11, 4, 2, 2, 4, 0, X, X, X, X, X, X, X}},
+       {{11, 7, 4, 11, 4, 2, 8, 3, 4, 3, 2, 4, X, X, X, X}},
+       {{2, 9, 10, 2, 7, 9, 2, 3, 7, 7, 4, 9, X, X, X, X}},
+       {{9, 10, 7, 9, 7, 4, 10, 2, 7, 8, 7, 0, 2, 0, 7, X}},
+       {{3, 7, 10, 3, 10, 2, 7, 4, 10, 1, 10, 0, 4, 0, 10, X}},
+       {{1, 10, 2, 8, 7, 4, X, X, X, X, X, X, X, X, X, X}},
+       {{4, 9, 1, 4, 1, 7, 7, 1, 3, X, X, X, X, X, X, X}},
+       {{4, 9, 1, 4, 1, 7, 0, 8, 1, 8, 7, 1, X, X, X, X}},
+       {{4, 0, 3, 7, 4, 3, X, X, X, X, X, X, X, X, X, X}},
+       {{4, 8, 7, X, X, X, X, X, X, X, X, X, X, X, X, X}},
+       {{9, 10, 8, 10, 11, 8, X, X, X, X, X, X, X, X, X, X}},
+       {{3, 0, 9, 3, 9, 11, 11, 9, 10, X, X, X, X, X, X, X}},
+       {{0, 1, 10, 0, 10, 8, 8, 10, 11, X, X, X, X, X, X, X}},
+       {{3, 1, 10, 11, 3, 10, X, X, X, X, X, X, X, X, X, X}},
+       {{1, 2, 11, 1, 11, 9, 9, 11, 8, X, X, X, X, X, X, X}},
+       {{3, 0, 9, 3, 9, 11, 1, 2, 9, 2, 11, 9, X, X, X, X}},
+       {{0, 2, 11, 8, 0, 11, X, X, X, X, X, X, X, X, X, X}},
+       {{3, 2, 11, X, X, X, X, X, X, X, X, X, X, X, X, X}},
+       {{2, 3, 8, 2, 8, 10, 10, 8, 9, X, X, X, X, X, X, X}},
+       {{9, 10, 2, 0, 9, 2, X, X, X, X, X, X, X, X, X, X}},
+       {{2, 3, 8, 2, 8, 10, 0, 1, 8, 1, 10, 8, X, X, X, X}},
+       {{1, 10, 2, X, X, X, X, X, X, X, X, X, X, X, X, X}},
+       {{1, 3, 8, 9, 1, 8, X, X, X, X, X, X, X, X, X, X}},
+       {{0, 9, 1, X, X, X, X, X, X, X, X, X, X, X, X, X}},
+       {{0, 3, 8, X, X, X, X, X, X, X, X, X, X, X, X, X}},
+       {{X, X, X, X, X, X, X, X, X, X, X, X, X, X, X, X}}}};
```

Comment:
I hope that you have good test because if there is a typo in these tables we will never be able to find it.

Reply:
I have not tested all configuration. I hope that the source did not do any mistakes^^